nixos/doc/manual/development/writing-modules.chapter.md

```diff
@@ -141,7 +141,8 @@ in {
         path  = [ pkgs.su ];
         script =
           ''
-            mkdir -m 0755 -p $(dirname ${toString cfg.output})
+            mkdir -p $(dirname ${toString cfg.output})
+            chmod 0755 $(dirname ${toString cfg.output})
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2174#correct-code

These two are still the same

Reply:
Yes, the idea was to do a refactor, not to change the functionality to the one potentially intended by the original author. Future commits could change the semantics of this code, if deemed necessary.

Reply:
So this is only to suppress warning here?

Reply:
And to highlight the issue that the original code did not do what it seemed to do, and could break if the output directory was several levels deep.